<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
  <head>
    <title>Proteomaps</title>
    <link rel="stylesheet" type="text/css" href="proteomaps.css">
    <script src="shiftzoom.js" language="javascript" type="text/javascript"></script>

  <script>
   (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
   (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
   m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
   })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
   ga('create', 'UA-4526009-11', 'proteomaps.net');
   ga('send', 'pageview');
  </script>

  </head>

  <body>
    <h1>How to browse proteomaps</h1>
    <table><tr>
	  <td>
	    <div> <a href="index.html"><img class="wrap"  src="pictures/eco_Valgepea_2013_48_cost_small.png" width="256"></a> </div>  
	  <td width="10"/>    
	  <td>    

    </tr>
    </table>

    <h3>What do the proteomaps show?</h3> 
    
    <p style="width:750">All maps show measured protein abundances,
      weighted by protein chain lengths. Links in the top line take you to
      the four hierarchy levels (level 1, 2, 3, and the protein level).<p/>
      
    <p style="width:750">Each protein or protein category occupies a
    certain area, i.e. a certain mass-certain fraction of the total
    proteome. To see the numerical values, move the mouse pointer over
    a region of interest: after a short while, the value will be
    visible. The mass abundance (measured abundance times protein
    chain length) is given as a percentage value. Currently, this
    feature is only available in some of the proteomaps.<br><br>

 The estimated copy (molecule) number is obtained by scaling the
 measured abundances to the total protein number per cell. We note
 that growth conditions can change cell volume several fold and thus
 affect the copy numbers accordingly. We use a generic value of ~3
 million proteins per micron cubed of cell volume (<a
 href="http://onlinelibrary.wiley.com/doi/10.1002/bies.201300066/abstract"
 target="_blank">Milo, Bioessays 2013</a>) and characteristic reported
 cell volumes. This leads us to the following total protein number per
 cell type:

</p>
  <table border = "1" style="background-color:#eeeeee">
      <tr><td><em>Mycoplasma pneumoniae</em></td><td> 5 * 10<sup>4</sup> molecules/cell</td></tr>
      <tr><td><em>Escherichia coli</em></td><td> 3 * 10<sup>6</sup> molecules/cell</td></tr>
      <tr><td><em>Saccharomyces cerevisiae</em></td><td> 1 * 10<sup>8</sup> molecules/cell</td></tr>
      <tr><td><em>Schizosaccharomyces pombe</em></td><td> 3 * 10<sup>8</sup> molecules/cell</td></tr>
      <tr><td><em>Arabidopsis thaliana</em></td><td> 1 * 10<sup>10</sup> molecules/cell</td></tr>
      <tr><td><em>Mus musculus</em></td><td> 1 * 10<sup>10</sup> molecules/cell</td></tr>
      <tr><td><em>Homo sapiens</em></td><td> 1 * 10<sup>10</sup> molecules/cell</td></tr>
    </table>

    <h3>Information in the protein-level maps</h3> 

    <p style="width:750">In the protein-level maps, systematic gene
    names or Uniprot IDs (as used inthe original proteome data) are
    shown in brackets, and most proteins are linked to pages from the
    <a href="http://www.genome.jp/kegg/">KEGG database</a>, showing
    detailed information about the protein. Click on a tile to access
    the corresponding page.</p>


    <h3>How to use the zoom</h3> 

   <p style="width:750">The "Zoom" version of the protein map has no
   links, but allows you to zoom in and out easily (using mouse wheel
   or page up/down keys) and to move within the map (using the cursor
   keys).</p>

<p>Back to <a href="index.html">Proteomaps home</a></p>

  </body>
</html>
